util/aws_tools.go

```diff
@@ -156,3 +165,142 @@ func ConstructCloudwatchInsightsQueryDetail(entity cloudwatchlogsTypes.QueryDefi
 	queryDetailBuilder.WriteString(awsOuterEscape("))"))
 	return strings.ReplaceAll(queryDetailBuilder.String(), "%", "$")
 }
+
+func GenerateLogoutLoginUrl(destinationUrl string) url.URL {
+	loginUrlParams := url.Values{}
+	loginUrlParams.Add("Action", "login")
+	loginUrlParams.Add("Issuer", "aws-vault")
+	loginUrlParams.Add("SigninToken", SigninToken)
+	loginUrlParams.Add("Destination", destinationUrl)
+
+	loginUrl := url.URL{
+		Scheme:   "https",
+		Host:     "us-east-1.signin.aws.amazon.com",
```

Comment:
here and below: will this work for other regions as well? it feels like we should either a) make this URL region-aware, or b) otherwise strip it if AWS handles that automatically (though if there are redirects that slows that down, we should instead just make the URL region aware)